src/core/components/modal/index.tsx

```diff
@@ -24,7 +27,7 @@ import styles from './styles';
  * Uses styles defined on the <picker-field> element for the modal and buttons.
  * This is used on iOS only.
  */
-export default (props: Props): Node => {
+export default (props: Props): JSX.Element => {
```

Comment:
Not ReactNode?

Reply:
TSC was complaining about this, I can try revisit

Reply:
<img width="624" alt="Screenshot 2023-10-05 at 9 55 29 AM" src="https://github.com/Instawork/hyperview/assets/309515/8526de61-26b4-4131-a830-ae71c8e18dd7">